test/k8sT/manifests/kafka-sw-security-policy.yaml

```diff
@@ -10,7 +10,18 @@ specs:
     ingress:
     - fromEndpoints:
       - matchLabels:
-          app: kafka
+          "reserved:host": ""
+      toPorts:
+      - ports:
+        - port: "9092"
+          protocol: TCP
+  - endpointSelector:
+      matchLabels:
+        app: kafka
+    egress:
+    - fromEndpoints:
```

Comment:
I know this is WIP, but this will fail - needs to be `toEndpoints` instead of `fromEndpoints`.

Reply:
ouch, thanks